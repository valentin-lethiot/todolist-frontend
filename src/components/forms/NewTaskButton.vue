<script setup lang="ts"></script>

<template>
  <div class="absolute right-8 bottom-8">
    <button
      class="rounded-full bg-gray-800 border-gray-700 hover:bg-gray-700 hover:cursor-pointer text-white text-5xl font-bold px-3 pb-3"
    >
      +
    </button>
  </div>
</template>
