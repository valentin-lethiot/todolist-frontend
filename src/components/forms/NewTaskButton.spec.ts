import { describe } from 'node:test'
import { beforeEach, expect, test } from 'vitest'
import NewTaskButton from './NewTaskButton.vue'
import { mount, VueWrapper } from '@vue/test-utils'

describe('NewTaskButton', () => {
  let wrapper: VueWrapper

  beforeEach(() => {
    wrapper = mount(NewTaskButton)
  })

  describe('render', () => {
    test('should render the button', () => {
      expect(wrapper.html()).toMatchSnapshot()
    })
  })
})
