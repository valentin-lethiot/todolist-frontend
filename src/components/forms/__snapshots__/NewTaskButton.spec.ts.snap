// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should render the button 1`] = `"<div class="absolute right-8 bottom-8"><button class="rounded-full bg-gray-800 border-gray-700 hover:bg-gray-700 hover:cursor-pointer text-white text-5xl font-bold px-3 pb-3"> + </button></div>"`;
